extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/SmallRyeHealthProcessor.java

```diff
@@ -52,10 +60,22 @@
     @ConfigRoot(name = "smallrye-health")
     static final class SmallRyeHealthConfig {
         /**
-         * The path of the health-checking servlet.
+         * The path of the legacy health-checking servlet.
          */
         @ConfigItem(defaultValue = "/health")
-        String path;
+        String healthPath;
+
+        /**
+         * The relative path of the liveness health-checking servlet.
+         */
+        @ConfigItem(defaultValue = "/live")
```

Comment:
I'm wondering if using `@ConfigItem(defaultValue = "${healthPath}/live")` would work?

If it's just the root path, I think it would be better.

Reply:
For me personally, having a health root path that affects this value is confusing. I would either leave it the way @antoinesd has it, or we really want to make things clear and don't really mind about 100% backward compatibility at this point, I would drop the `healthPath` field completely.

Reply:
So my opinion is that we should drop exposing the servlet on the legacy `healthPath`.

So `healthPath` would be `rootPath`. I'm also not convinced it has any interest to have it instead of just `readinessPath` and `livenessPath`. Not sure if they plan to add more probes in MP Health. In this case, it could make sense.

Reply:
@kenfinnigan you were the one asking for a root path. Do you think it has some strong value for the future?

Reply:
I didn't know it was possible to write this. Of course, it's far better to go that way

Reply:
@gsmet my thought with a root path was that then it would be possible to change the path for everything under /health in one setting, instead of needing to change to paths